targets/TARGET_Maxim/TARGET_MAX32600/device/TOOLCHAIN_IAR/MAX32600.icf

```diff
@@ -3,9 +3,7 @@ define symbol __intvec_start__        = 0x0;
 define symbol __region_ROM_start__    = 0x0;
 define symbol __region_ROM_end__      = 0x0003FFFF;
 
-/* [RAM] Vector table dynamic copy: 79 vectors * 4 bytes = 316 bytes (0x13C) */
-define symbol __NVIC_start__          = 0x00000000;
-define symbol __NVIC_end__            = 0x00000140; /* to be aligned on 8 bytes */
+/* [RAM] */
```

Comment:
vector size was 79*4=0x13C (not aligned to 8-byte) 
Was updated to be 8-byte aligned as 0x140. I am not sure why are we removing vector region from RAM ?